CHANGELOG.md

```diff
@@ -22,3 +19,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Security
 
 ## [1.0.0] - 2021.12.12
+
+## [1.1] - 2021.12.24
```

Comment:
So, not to seem pedantic, but when making a release the general flow of things for the changelog is:
1. Copy the unreleased section, including the subheaders/change categories. Paste it above the current unreleased section.
2. Update the old `[Unreleased]` header to `[#.#.#] - YYYY.MM.DD`, reflecting the new version (I think we should always use the three digit format for releases even if the last digit is a 0. Just for consistency sake, but we can talk about that.)
3. Delete any unused category headers under the new release and delete any copied changes that went into the new unreleased section.

The latest version should always be at the top, under the unreleased section. This is for readability. Your changelog change should look more like this:
```
[Unreleased]
...
[1.1.0] - 2021.12.24

### Added

- Added GitHub templates for pull request and pull request review.
 - Rewrote README with the goal of making the workflow of this project clear to future developers.
 - Merged MinePi project (custom functionality for running this project on Raspberry Pi). When running this project on a Raspberry Pi, the project now monitors the system's CPU temperature and shuts down if it detects a critical event.

[1.0.0] - 2021.12.24
...
```

If this doesn't make sense, we can have a real quick call to discuss this in further detail.

Reply:
Updated.